.github/CONTRIBUTING.md

```diff
@@ -30,3 +33,50 @@ This pull request will be a perfect start to find the fix :)
   
   If your pull request involves fixing SonarQube issues then we would suggest that you please discuss this with the 
   [TestNG-dev](https://groups.google.com/forum/#!forum/testng-dev) before you spend time working on it.
+
+# Working with the codebase
+
+### List all tasks
+
+After cloning the project, run `./gradlew tasks` to list out all the tasks that are available.
+
+### Importing into IntelliJ
+
+To import the TestNG project into IntelliJ run `./gradlew openIdea`
```

Comment:
What is the value-add of the idea plugin? I always import my projects from the IntelliJ menu.

In fact, I think the IDE part is not important because IntelliJ is not mandatory for TestNG and we don't have a custom configuration.

Reply:
I always have problems with idea and the codebase. The plugin does let me use some of the tasks to manage this. Yes I agree that the codebase doesn't have affinity to any IDE but I don't think it hurts to include this info as well.

Reply:
It is not a problem to add the plugin but what are the problems you have?

Reply:
@juherr - The most common problem that I have is when I switch between branches, the IDE goes corrupt and I start seeing compilation errors. They dont get fixed even after I do a clear cache or click on the "Reload all projects" option from Gradle 

![image](https://user-images.githubusercontent.com/909773/130427591-33b15042-8e86-458e-a0bd-dd29b9aa34df.png)

The worst part is, they sometimes work and sometimes dont. Found the idea plugin to be highly effective in ensuring that IntelliJ never complains.

Reply:
Ok :+1: